cgi-bin/LJ/S2Theme/headsup.pm

```diff
@@ -17,9 +17,10 @@ sub header_props {
     my $self = shift;
     my @props = qw(
         image_foreground_header_url
-        int image_foreground_header_height
-        string image_foreground_header_alignment
+        image_foreground_header_height
```

Comment:
Forgive me for taking so long to review this - I'm not as familiar with the conventions here.  Why was the "int" removed from the list here - does it make a difference at all?